config/prod/tom-cvat-alb.yaml

```diff
@@ -0,0 +1,24 @@
+# Provision a load balancer for minidream.synapse.org
+template_path: remote/managed-alb-https.yaml
+stack_name: tom-cvat-alb
+parameters:
+  # The Sage deparment (Platform, CompOnc, SysBio, Governance, etc..)
+  Department: "CompOnc"
+  # The Sage project (Infrastructure, amp-ad, ntap, dream, etc..)
+  Project: "dream"
+  # The resource owner
+  OwnerEmail: "thomas.yu@sagebase.org"
+
+   # load balancer configs
+  Ec2InstanceId: "i-04c318988a159582a"
+  SSLCertificateIdArn: "arn:aws:acm:us-east-1:055273631518:certificate/992c2d49-579a-465e-beff-1a08c284f7db"
```

Comment:
the "*.synapse.org" certificate in sandbox is "arn:aws:acm:us-east-1:563295687221:certificate/b7f3a44b-1c72-4f84-8e8a-42050764e6c2"